src/main/java/com/configcat/ConditionAccessor.java

```diff
@@ -0,0 +1,12 @@
+package com.configcat;
+
+/**
+ * The interface for the data classes which can be passed to the RolloutEvaluator.evaluateConditions method.
+ */
+public interface ConditionAccessor {
```

Comment:
Couldn't this interface be package-private? That way, I'd also omit the docs comment.